cassandra-4.0/Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM openjdk:8u252-slim
 
-ARG STARGATE_VERSION=v1.0.0
+ARG STARGATE_VERSION=v1.0.1
 
-RUN apt update && apt install curl unzip iproute2 libaio1 -y
-RUN curl -sL https://github.com/stargate/stargate/releases/download/${STARGATE_VERSION}/stargate-jars.zip > stargate-jars.zip
+RUN apt update -qq \
+    && apt install curl unzip iproute2 libaio1 -y \
+    && apt autoremove --yes \
+    && apt clean all \
+    && rm -rf /var/lib/{apt,dpkg,cache,log}/
```

Comment:
Would it make sense to tag a local image at this point and reuse it for C* version-dependent images? Even if it does not help the size it should allow us to consolidate repeated `Dockerfile` commands and will probably reduce built time.

Reply:
done